src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -11,160 +11,160 @@
-      "Size": 3645
+      "Size": 3651
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 398294
+      "Size": 398300
     },
     "assemblies/mscorlib.dll": {
-      "Size": 3832
+      "Size": 3830
     },
     "assemblies/netstandard.dll": {
-      "Size": 5521
+      "Size": 5523
     },
     "assemblies/rc.bin": {
       "Size": 802
     },
     "assemblies/System.Collections.Concurrent.dll": {
-      "Size": 11579
+      "Size": 11584
     },
     "assemblies/System.Collections.dll": {
-      "Size": 19185
+      "Size": 19142
     },
     "assemblies/System.Collections.NonGeneric.dll": {
-      "Size": 8471
+      "Size": 8478
     },
     "assemblies/System.ComponentModel.dll": {
-      "Size": 2002
+      "Size": 2006
     },
     "assemblies/System.ComponentModel.Primitives.dll": {
-      "Size": 2607
+      "Size": 2612
     },
     "assemblies/System.ComponentModel.TypeConverter.dll": {
-      "Size": 6993
+      "Size": 7003
     },
     "assemblies/System.Console.dll": {
-      "Size": 5839
+      "Size": 5843
     },
     "assemblies/System.Core.dll": {
-      "Size": 1968
+      "Size": 1970
     },
     "assemblies/System.Diagnostics.TraceSource.dll": {
-      "Size": 6800
+      "Size": 6806
     },
     "assemblies/System.dll": {
-      "Size": 2315
+      "Size": 2318
     },
     "assemblies/System.Drawing.dll": {
-      "Size": 2000
+      "Size": 2003
     },
     "assemblies/System.Drawing.Primitives.dll": {
-      "Size": 12154
+      "Size": 12158
     },
     "assemblies/System.Formats.Asn1.dll": {
       "Size": 26856
     },
     "assemblies/System.IO.Compression.Brotli.dll": {
-      "Size": 11461
+      "Size": 11464
     },
     "assemblies/System.IO.Compression.dll": {
-      "Size": 18810
+      "Size": 18818
     },
     "assemblies/System.IO.FileSystem.dll": {
       "Size": 1964
     },
     "assemblies/System.IO.IsolatedStorage.dll": {
-      "Size": 10628
+      "Size": 10630
     },
     "assemblies/System.Linq.dll": {
-      "Size": 19504
+      "Size": 19510
     },
     "assemblies/System.Linq.Expressions.dll": {
       "Size": 181323
     },
     "assemblies/System.Net.Http.dll": {
-      "Size": 211166
+      "Size": 211322
     },
     "assemblies/System.Net.NameResolution.dll": {
-      "Size": 9924
+      "Size": 9931
     },
     "assemblies/System.Net.NetworkInformation.dll": {
-      "Size": 17325
+      "Size": 17336
     },
     "assemblies/System.Net.Primitives.dll": {
-      "Size": 41156
+      "Size": 41162
     },
     "assemblies/System.Net.Quic.dll": {
-      "Size": 43453
+      "Size": 43606
     },
     "assemblies/System.Net.Security.dll": {
-      "Size": 57317
+      "Size": 57359
     },
     "assemblies/System.Net.Sockets.dll": {
-      "Size": 54458
+      "Size": 54449
     },
     "assemblies/System.ObjectModel.dll": {
-      "Size": 11316
+      "Size": 12003
     },
     "assemblies/System.Private.CoreLib.dll": {
-      "Size": 698582
+      "Size": 753071
```

Comment:
Noticed this file is ~50kb larger in the Xamarin.Forms app, but we don't see the same different in the "Hello World" Android app.

Reply:
Weird. Could it be that we are getting different architectures?

Reply:
Or just the linker is keeping more with Xamarin.Forms.